CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-Welcome to MayaData. We are excited about the prospect of you joining us! We abide by the values outlined in [PLOW](https://github.com/mayadata-io/culture/blob/master/plow.md), which is a superset of generally accepted [code of conduct](CODE_OF_CONDUCT.md).
+Welcome to MayaData. We are excited about the prospect of you joining us! We abide by the values outlined in [PLOW](./plow/plow.md), which is a superset of generally accepted [code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
Most of the places use absolute url. Suggesting just that I like consistency if possible. If it has a reason, feel free to ignore.
```suggestion
Welcome to MayaData. We are excited about the prospect of you joining us! We abide by the values outlined in [PLOW](/plow/plow.md), which is a superset of generally accepted [code of conduct](CODE_OF_CONDUCT.md).
```